cargo-near-build/src/env_keys.rs

```diff
@@ -35,12 +37,21 @@ pub mod nep330 {
     }
 
     pub(crate) fn print_env() {
-        tracing::info!("Variables, relevant for reproducible builds:");
+        let mut env_map: HashMap<&str, String> = HashMap::new();
         for key in [BUILD_ENVIRONMENT, CONTRACT_PATH, SOURCE_CODE_SNAPSHOT] {
             let value = std::env::var(key)
                 .map(|val| format!("'{}'", val))
                 .unwrap_or("unset".to_string());
-            tracing::info!("{}={}", key, value);
+            env_map.insert(
+                key,
+                value.replace("'", "").replace("\"", "").replace(",", " "),
```

Comment:
rust's debug representation `"{:#?}"` should be probably used as is here (and wrap in `'...'` single quotes on line 43 removed too)

Reply:
Done